pam_fscrypt/run_fscrypt.go

```diff
@@ -67,6 +68,19 @@ type PamFunc struct {
 	impl func(handle *pam.Handle, args map[string]bool) error
 }
 
+// isSystemUser checks if a user is a system user.  pam_fscrypt should never
+// need to do anything for system users since they should never have login
+// protectors.  Therefore, we detect them early to avoid wasting resources.
+func isSystemUser(user *user.User) bool {
+	uid := util.AtoiOrPanic(user.Uid)
+	if uid > 0 && uid < 1000 {
```

Comment:
nit: could use `SYS_UID_MAX` here

Reply:
That would require reading and parsing `/etc/login.defs`, right?  If that is the case, then I'm not sure we should bother with this optimization in the first place.  My hope was that the 1-999 range is standard enough that we can assume it.

Reply:
We should just have a constant defined `UID_MIN = 1000` and check that we have `uid >= UID_MIN`. If it ever becomes an issue we can fix stuff later.

It seems like 1000 is the default, and Linux distros don't really change it: https://serverfault.com/questions/804718/how-to-determine-sys-uid-min-sys-uid-max-etc-values-from-a-script

Reply:
Added a constant.